MinecraftClient/ChatBot.cs

```diff
@@ -220,7 +220,18 @@ private McClient Handler
         /// <param name="locked"></param>
         /// <param name="iconcount"></param>
         public virtual void OnMapData(int mapid, byte scale, bool trackingposition, bool locked, int iconcount) { }
-        
+
+        /// <summary>
+        /// Called when tradeList is received from server
+        /// </summary>
+        /// <param name="windowID">Window ID</param>
+        /// <param name="trades">List of trades.</param>
+        /// <param name="villagerLevel">The level the villager is.</param>
+        /// <param name="experience">The amount of experience the villager has.</param>
+        /// <param name="isRegularVillager">True if regular villagers and false if the wandering trader.</param>
+        /// <param name="canRestock">If the villager can restock his trades at a workstation, True for regular villagers and false for the wandering trader.</param>
+        public virtual void OnTradeList(int windowID, List<Trade> trades, int villagerLevel, int experience, bool isRegularVillager, bool canRestock) { }
```

Comment:
```
public virtual void OnTradeList(int windowID, List<Trade> trades, VillagerInfo villagerInfo) { }
public class VillagerInfo
{
    public int Level { get; set; }
    public int Experience { get; set; }
    public bool IsRegularVillager { get; set; }
    public bool CanRestock { get; set; }
}
```

I can't figure out how to get the actual value into the set part. What I could do is add a dataType just like trade but that is probably not what you intended right?